packages/clients/meldemichel/src/mapConfigurations.ts

```diff
@@ -161,15 +164,17 @@ const mapConfigurations = {
             // TODO doesn't work atm; no coordinate source
             geometry: false,
             window: true,
-            properties: {
-              filename: 'Name of file',
-            },
+            properties: { filename: 'Name of file' },
           },
         },
         coordinateSources: [], // to be set in addPlugins.ts
       },
       pins: commonPins,
       reverseGeocoder,
+      meldemichelAfmButton: {
+        displayComponent: true,
+        afmUrl,
+      },
```

Comment:
```suggestion
      meldemichelAfmButton: { afmUrl },
```
`displayComponent` already defaults to `true` as set in `packages/clients/meldemichel/src/polar-client.ts`

Reply:
https://github.com/Dataport/polar/pull/23/commits/d29bd8fb7d771552f4b08fa652f8f8ea539d5274